internal/objects/service/service_test.go

```diff
@@ -138,6 +138,7 @@ func TestDesiredEnvoyService(t *testing.T) {
 		SpecNs:      "projectcontour",
 		RemoveNs:    false,
 		NetworkType: operatorv1alpha1.NodePortServicePublishingType,
+		NodePorts:   operatorv1alpha1.MakeNodePorts(map[string]int{"http": 30080, "https": 30443}),
```

Comment:
maybe use nonstandard ports in the test to make sure they are set properly? seems like the rest of the test didn't have to change which could mean the defaults were still in use in the implementation